users/ritvikkolhe/Task 3/Quiz.js

```diff
@@ -1,7 +1,24 @@
 var count=0
 function add(){
-  count=count+1;
+  var Q1 = document.getElementsByName('Q1').value;
+  var Q2 = document.getElementsByName('Q2').value;
+  var Q3 = document.getElementsByName('Q3').value;
+  var Q4 = document.getElementsByName('Q4').value;
+  var Q5 = document.getElementsByName('Q5').value;
+  console.log(Q1);
+  console.log(Q2);
+  if(Q1=='Correct')
+  count+=1;
+  if(Q2=='Correct')
+  count+=1;
+  if(Q3=='Correct')
+  count+=1;
+  if(Q4=='Correct')
+  count+=1;
+  if(Q5=='Correct')
+  count+=1;
```

Comment:
A better way to do this would have been
```
var ques = [];
for(i = 0;i < 5; i++){
    ques[i] = document.getElementsByName('Q' + str(i)).value;
    if(ques[i]  == 'Correct')count += 1;
}
```

Jus tkeep this in mind ;)